Adds DIS Pluto, First Hull of the Drake Industries Shipyards

UnicornOnLSD opened the pull request:
<!-- Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline -->

## About the PR
<!-- What did you change? -->
<!-- If this is a code change, summarize at high level how your new code works. This makes it easier to review. -->
Added the DIS Pluto. An (ideally) exclusive ship to the Drake Industries corporation. As of now it's adding the ship because whynot, if in the future such thing exist, then so be it.

Price: 29900
8 thrusters, 2 gyros, reaches top speed going forward in 10 seconds.
FTL capable (model 25)
12 base cargo crate space with a customisable bay for retrofitters.

I will likely add more flavor stuff in the future, once company got a logo and the such to make it stand out to the regular civilian ships.

## Why / Balance
<!-- Discuss how this would affect game balance or explain why it was changed. Link any relevant discussions or issues. -->
Drake Industries got recently added a a corporation. One of the aim to make it a bit more "alive" in the game is having specific hulls that brings it's spirit in game.

The aim of these ships are to be cargo focused, very compact and capable designed, ideally restricted to members of the corporation. For now it's just like any ship
## How to test
<!-- Describe the way it can be tested -->
Load dev env

mapping [grid number] /Maps/_Mono/pluto.yml true

Check it out yourself

## Media
<!-- Attach media if the PR makes ingame changes (clothing, items, features, etc). 
Small fixes/refactors are exempt. Media may be used in SS14 progress reports with credit. -->

![image](https://github.com/user-attachments/assets/32ab1ddc-69ec-4bf6-8b19-eb66f2b9cafa)
![Plutoad](https://github.com/user-attachments/assets/22206910-eb24-4969-a388-ac1500934f81)

## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.
- [X] I can confirm this PR contains no AI-generated content, and did not use any AI-generated content.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->

## Breaking changes
<!-- List any breaking changes, including namespaces, public class/method/field changes, prototype renames; and provide instructions for fixing them.
This will be posted in #codebase-changes. -->
not that I know of. 
**Changelog**
<!-- Add a Changelog entry to make players aware of new features or changes that could affect gameplay.
Make sure to read the guidelines and take this Changelog template out of the comment block in order for it to show up.
Changelog must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog. -->

:cl:
- add: Added the DIS Pluto. A small compact,  FTL capable cargo hauler straight from DIS.

UnicornOnLSD committed:
added ship and file NEED NAME STILL

monolith8319 committed:
chore: automatically update REUSE headers

UnicornOnLSD committed:
gave name and full desc

UnicornOnLSD committed:
Merge branch 'DIS-NAME-PENDING' of https://github.com/UnicornOnLSD/Monolith into DIS-NAME-PENDING

UnicornOnLSD committed:
Merge branch 'Monolith-Station:main' into DIS-NAME-PENDING

UnicornOnLSD commented:
Also made the cargo area decals sprayable with space cleaner, for all the retrofit fans out there who want to turn it into whatever.

UnicornOnLSD committed:
lowered price to 33000

UnicornOnLSD committed:
Merge branch 'DIS-NAME-PENDING' of https://github.com/UnicornOnLSD/Monolith into DIS-NAME-PENDING

monolith8319 committed:
chore: automatically update REUSE headers

UnicornOnLSD commented:
reduced price from 49 500 to 33 000 (double the appraisal value)

It's similar to the twilight except much less versatile and smaller.
Should definitely be restricted to corporate specific in the future for flavor's sake. As of now with this price fits the balance somewhat okay.

Redrover1760 reviewed (changes requested):
this will fix your linter

also if there is nothing special going on with a ship that would warrant a price increase you should set your ship's price to slightly above appraisal value. 16483 -> 17525 or something

Redrover1760 commented on Resources/Prototypes/_Mono/Shipyard/pluto.yml:
```diff
@@ -0,0 +1,57 @@
+  parent: BaseVessel
+  name: DIS Pluto
+  description: A frigate sized cargo hauler manufactured by the Drake Industries shipyards. Fitted with a 25 type bluespace drive and more than enough space for small scale cargo hauling.
+  price: 33 000
```
```suggestion
  price: 33000
```

Redrover1760 replied:
^ this will fix your linter I mean

UnicornOnLSD commented:
> this will fix your linter
> 
> also if there is nothing special going on with a ship that would warrant a price increase you should set your ship's price to slightly above appraisal value. 16483 -> 17525 or something

Taken into account, will do along with some more changes.

Thanks for the linter fix.

UnicornOnLSD committed:
Lowered price significantly, fixed linter error. Added wallmount locker instead.

UnicornOnLSD committed:
Merge branch 'DIS-NAME-PENDING' of https://github.com/UnicornOnLSD/Monolith into DIS-NAME-PENDING

monolith8319 committed:
chore: automatically update REUSE headers

UnicornOnLSD commented:
moved it to 19 900 and fixed linter issue.

It's main selling point is fast moving ability and cheap hyperspace drive that's included with it, mainly why I had it that high (so people don't just buy the ship cheaply for it's FTL drive)

Redrover1760 commented:
fair

also, your ship has initialized entities on it according to the map schema error with mapinit

UnicornOnLSD commented:
AOUGH rolling it back give me a sec to fix that.

UnicornOnLSD committed:
fixed mapping error

UnicornOnLSD committed:
Merge branch 'DIS-NAME-PENDING' of https://github.com/UnicornOnLSD/Monolith into DIS-NAME-PENDING

UnicornOnLSD commented:
Fixxed mapping error, something to do with a frontier tagged wall locker, never touching that thing again.

Redrover1760 commented:
cursed

Redrover1760 commented on Resources/Prototypes/_Mono/Shipyard/pluto.yml:
```diff
@@ -0,0 +1,57 @@
+  mapPath: /Maps/_Mono/Shuttles/pluto.yml
+  minPlayers: 0
+  stations:
+    Twilight:
```
```suggestion
    Pluto:
```
This should be the ID of your BecomesStation component ID in your ship

Redrover1760 commented on Resources/Maps/_Mono/Shuttles/pluto.yml:
```diff
@@ -0,0 +1,1845 @@
+  2: FloorTechMaint
+  3: Lattice
+  0: Plating
+entities:
```
Your map lacks a BecomesStation Component

In order to make one, load up the map unitialized as usual, vv edit the grid, go to server components tab, add compmonet -> BecomesStation

Then add an ID to use for your shipyard prototype, "Pluto" would be an example of one. The frontier videos also explain how to do this

Redrover1760 replied:
Your pipe networks need to be colored, see https://docs.google.com/document/d/1RZk5MiTv6lh3V9r2Q1PSR7L1acvSDKEnBX2acV3y11w/edit?tab=t.7yjmhvwe1nfp for how to do it

UnicornOnLSD replied:
In the rollback when I sought to fix the map init issue I had to take a previous save which undid that part, will fix soonTM

Redrover1760 commented on Resources/Prototypes/_Mono/Shipyard/pluto.yml:
```diff
@@ -0,0 +1,57 @@
+  parent: BaseVessel
+  name: DIS Pluto
+  description: A frigate sized cargo hauler manufactured by the Drake Industries shipyards. Fitted with a 25 type bluespace drive and more than enough space for small scale cargo hauling.
+  price: 19900
```
Appraisegrid states 21921 spesos, not your fault, appraisegrid on uninitalized grids underestimates the value substantially

UnicornOnLSD replied:
Will move price up to 25 900.

UnicornOnLSD replied:
adjusted price to 29500. It's meant to be ABOVE the purchase price of a regular drive at CO (25K) and the excess 4500 accounts for the ship's optimisation, manoeuverability, given gear and cargo space.

UnicornOnLSD committed:
Update Resources/Prototypes/_Mono/Shipyard/pluto.yml

Co-authored-by: Redrover1760 <39284090+Redrover1760@users.noreply.github.com>

UnicornOnLSD committed:
added pipe coloring, adjusted price, added station comp and proper ID, adjusted ship desc, added ship notes.

UnicornOnLSD commented:
Final version all should be good now.

![image](https://github.com/user-attachments/assets/32ab1ddc-69ec-4bf6-8b19-eb66f2b9cafa)
![image](https://github.com/user-attachments/assets/b47e4691-cc93-4ab5-aea3-d8494067f46b)

Added become station comp, colored pipes,

monolith8319 committed:
chore: automatically update REUSE headers

UnicornOnLSD committed:
minute change to ship, see comment

monolith8319 committed:
chore: automatically update REUSE headers

UnicornOnLSD commented:
![image](https://github.com/user-attachments/assets/55bfce61-1002-47d8-8509-cc7ae9306d05)
More changes after checking out other cargo ships. QoL stuff highlithed all in picture.

UnicornOnLSD committed:
proper adjustment, last change didn't save it seems

UnicornOnLSD committed:
Merge branch 'DIS-NAME-PENDING' of https://github.com/UnicornOnLSD/Monolith into DIS-NAME-PENDING

UnicornOnLSD commented:
Ready for review, I swear this time.

UnicornOnLSD committed:
Merge branch 'Monolith-Station:main' into DIS-NAME-PENDING

UnicornOnLSD committed:
see comment

UnicornOnLSD commented:
![image](https://github.com/user-attachments/assets/364066b7-2b0d-46f2-a34a-8a57946be53b)

last minute update of things I realised I missed

the railing next to the conveyor are now set on the right tiles in order not to block the cargo rolling in
Made the gen into a ship var (p sure it wasn't before)
added vac markers on lattice and the such

monolith8319 committed:
chore: automatically update REUSE headers

UnicornOnLSD committed:
Merge branch 'Monolith-Station:main' into DIS-NAME-PENDING
